man/systemd-pcrphase.service.xml

```diff
@@ -67,33 +67,36 @@
       <listitem><para><literal>enter-initrd</literal> — early when the initrd initializes, before activating
       system extension images for the initrd. It acts as a barrier between the time where the kernel
       initializes and where the initrd starts operating and enables system extension images, i.e. code
-      shipped outside of the UKI. (This extension happens when
-      <filename>systemd-pcrphase-initrd.service</filename> is started.)</para></listitem>
+      shipped outside of the UKI. (This extension happens when the
+      <citerefentry><refentrytitle>systemd-pcrphase-initrd.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>
+      service is started.)</para></listitem>
 
       <listitem><para><literal>leave-initrd</literal> — when the initrd is about to transition into the host
-      file system. It acts as barrier between initrd code and host OS code. (This extension happens when
-      <filename>systemd-pcrphase-initrd.service</filename> is stopped.)</para></listitem>
+      file system. It acts as barrier between initrd code and host OS code. (This extension happens when the
+      <filename>systemd-pcrphase-initrd.service</filename> service is stopped.)</para></listitem>
```

Comment:
I don't think such kind of redundancy is really needed?

Reply:
Please explain.

Reply:
"systemd-pcrphase-initrd.service is stopped" is already clear enough. IMO the extra "service" doesn't make things better.

Reply:
For us, it's obvious that `foo.service` is a service. But for a casual reader, this might not be obviouus. When introducing the object, it's good to specify its type, even if it can be deduced from the name. Hence, "the foo* object" is used once, and then just "foo*" if it were referenced again.